MaxText/inference_mlperf/trillium/benchmarks_llama2-70b-trillium_2x4.sh

```diff
@@ -58,8 +58,14 @@ fi
 
 export TOK_OUTLEN_MULTIPLIER=${token_multiplier}
```

Comment:
We also want to enable accuracy runs - so either just uncomment in this script or add an option for it.